src/main/scala/com/fulcrumgenomics/umi/ConsensusTags.scala

```diff
@@ -48,6 +48,14 @@ object ConsensusTags {
   /** Post-grouping ID that is file-wide unique per source molecule. */
   val MolecularId = "MI"
 
+  /** The suffix appended to molecular identifier reads from the "top strand" of the duplex molecule. These reads have
+    * 5' coordinate for read 1 less than or equal to the 5' coordinate for read 2. */
+  val TopStrandDuplex = "/A"
```

Comment:
Everything else in here is a two-letter SAM attribute name.  It feels weird to put these constants here.  Any objection to just having them be public vals in `GroupReadsByUmi`'s companion object?